MessagesView/MessagesInputToolbar.swift

```diff
@@ -8,19 +8,34 @@
 
 import UIKit
 
-class MessagesInputToolbar: UIToolbar {
+class MessagesInputToolbar: UIView {
 
     let toolbarContentView = MessagesToolbarContentView.fromNib()
     
-    /*
-    // Only override draw() if you perform custom drawing.
-    // An empty implementation adversely affects performance during animation.
-    override func draw(_ rect: CGRect) {
-        // Drawing code
-    }
-    */
-    var messageText : String {
-        return toolbarContentView.messageText
+    var leftButtonAction: ()->() {
+        get {
+            return toolbarContentView.leftButtonAction
+        }
+        set {
+            toolbarContentView.leftButtonAction = newValue
+        }
+    }
+    var rightButtonAction: ()->() {
+        get {
+            return toolbarContentView.rightButtonAction
+        }
+        set {
+            toolbarContentView.rightButtonAction = newValue
+        }
+    }
+    
+    var inputText : String {
+        get {
+            return toolbarContentView.inputText
+        }
+        set {
+            toolbarContentView.inputText = newValue
+        }
```

Comment:
what is the purpose of explicit dividing getters and setters here?

Reply:
You tell me ;)  Commit: d007b8e